images/busybox/README.md

```diff
@@ -13,10 +13,43 @@
 
 Container image with only busybox and libc (available in both musl and glibc variants). Suitable for running any binaries that only have a dependency on glibc/musl.
 
+- [Documentation](https://edu.chainguard.dev/chainguard/chainguard-images/reference/busybox)
+- [Getting Started Guide](https://edu.chainguard.dev/chainguard/chainguard-images/reference/busybox/getting-started-php/)
+- [Provenance Information](https://edu.chainguard.dev/chainguard/chainguard-images/reference/busybox/provenance_info/)
+
+## Image Variants
+
+There are two variants, one for musl and one for gclib.
+
+- `latest`: This is a image that has busybox and is for musl based variants.
+- `latest-glibc`: This is a image that has busybox and is for glibc based variants.
+
 ## Get It!
 
 The image is available on `cgr.dev`:
 
-```
+```shell
 docker pull cgr.dev/chainguard/busybox:latest
 ```
+
+## Test it
```

Comment:
Maybe this can be `## Usage` to be consistent with others, as in https://github.com/chainguard-images/images/blob/main/images/curl/README.md#usage

```suggestion
## Usage
```